src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -677,6 +676,9 @@ private async Task CreateDeploymentBundle(Orchestrator orchestrator, Recommendat
         {
             if (selectedRecommendation.Recipe.DeploymentBundle == DeploymentBundleTypes.Container)
             {
+                _orchestratorInteractiveService.LogSectionStart("Creating deployment image",
+                    "Using the docker CLI perform a docker build to create a container image.");

```

Comment:
nit: Grammar here took me a few reads to parse this sentence
* "Using the docker CLI **to** perform...
* Using the docker CLI, perform a... (add the comma?)

(same comment down on line 716)

Reply:
It looks like `Orchestrator` was updated in the force-push but not `DeployCommand`